package.json

```diff
@@ -20,7 +20,7 @@
     "updated": "lerna updated --only-explicit-updates",
     "prettier": "node prettier.js",
     "clean": "lerna clean --yes && rimraf .jest",
-    "publish": "git pull --rebase && check-installed-dependencies && npm run build && lerna publish --only-explicit-updates",
+    "publish": "git pull --rebase && check-installed-dependencies && npm run build && lerna publish",
```

Comment:
I didn't review this change initially, I agree with @wyuenho, let's be explicit in what we publish